### ISSUE TITLE
Might be down

### ISSUE THREAD

**Issue by aleksa-codes** [digitallyinduced/thin-backend#76]
Hi everyone,

I know this project is not currently maintained, but I deployed one of my first decent projects while still learning web dev using Thin, so I'm interested in understanding whether Thin has been shut down or if there is a bug or internal error. This information will help me decide whether to retire my project, which is part of my portfolio.

When I go to the "Data" section in the dashboard, I encounter the following error:
![image](https://github.com/digitallyinduced/thin-backend/assets/64015178/ad234e02-1747-4b29-a4da-1e9453036496)

Additionally, when trying to log in to the project, I see this error:
![image](https://github.com/digitallyinduced/thin-backend/assets/64015178/3d9b381f-136f-4d56-987e-0da3f3e69f09)

I loved Thin; it was a great idea and project. I wish you all the best!

**Comment by mpscholten** [comment]
Yes we shut down the server for now as it's too expensive to run (and we ran out of free aws credits)
